sql/migrate/migrate.go

```diff
@@ -381,7 +381,7 @@ func (d *LocalDir) Open(name string) (fs.File, error) {
 
 // WriteFile implements Dir.WriteFile.
 func (d *LocalDir) WriteFile(name string, b []byte) error {
-	return os.WriteFile(filepath.Join(d.dir, name), b, 0600)
+	return os.WriteFile(filepath.Join(d.dir, name), b, 0644) // nolint: gosec
```

Comment:
move this config to golangci.yml

Reply:
Maybe there's a cleaner configuration for it, but here's how I did it in ent https://github.com/ent/ent/blob/master/.golangci.yml#L60-L62